airbyte-integrations/connectors/source-facebook-marketing/source_facebook_marketing/api.py

```diff
@@ -161,7 +161,7 @@ class API:
     def __init__(self, account_id: str, access_token: str):
         self._account_id = account_id
         # design flaw in MyFacebookAdsApi requires such strange set of new default api instance
-        self.api = MyFacebookAdsApi.init(access_token=access_token, crash_log=False)
+        self.api = MyFacebookAdsApi.init(access_token=access_token, crash_log=False, api_version='v14.0')
```

Comment:
You need to change the facebook sdk version in setup.py. Please sign the CLA also and bump the connector version in the dockerfile and make sure you allow maintainer edit your PR.

Reply:
SDK version was not changed. Still 13.0.0 but API version is 14.0 @marcosmarxm